src/TodoListItem.js

```diff
@@ -1,7 +1,18 @@
 // import {react} from 'react'
 
+import React from "react";
+
 function TodoListItem(props) {
-  return <li>{props.item}</li>;
+const onRemoveTodo=()=>{
+  // props.onRemoveTodo(props.item.id)
```

Comment:
Hey Juliette! I think some our conversation got lost in translation. This line has to be uncommented out. The remove functionality doesn't currently work : (.